plugins/be/src/src.pro

```diff
@@ -0,0 +1,47 @@
+TOP_BUILDDIR = $$OUT_PWD/../../..
+TOP_SRCDIR = $$PWD/../../..
+
+include($${TOP_SRCDIR}/config.pri)
+
+TEMPLATE        = lib
+CONFIG         += plugin
+QT             += widgets
+INCLUDEPATH    += \
+    $${TOP_SRCDIR}/src/ \
+    $${TOP_SRCDIR}/src/lib/ \
+    $${TOP_SRCDIR}/src/lib/logic/
+    $${TOP_SRCDIR}/plugins/westernsupport
+
+HEADERS         = \
+    belarusianplugin.h
+
+TARGET          = $$qtLibraryTarget(beplugin)
+
+EXAMPLE_FILES = belarusianplugin.json
+
+# generate database for presage:
+PLUGIN_INSTALL_PATH = $${UBUNTU_KEYBOARD_LIB_DIR}/be/
+
+lang_db_ru.commands += \
+  rm -f $$PWD/database_be.db && \
```

Comment:
It appears that this database is not installed correctly, causing the keyboard to crash when this layout is loaded:

```
WARNING: Did not find a dictionary for "be"  - checking for  "be"
WARNING: No dictionary found for "be" turning off spellchecking
WARNING: no dictionary to turn on spellchecking
[DatabaseConnector] Unable to create/open database: /usr/share/maliit/plugins/com/ubuntu/lib/be/database_be.db
terminate called after throwing an instance of 'SqliteDatabaseConnector::SqliteDatabaseConnectorException'
  what():  unable to open database file
```

This may be related to using `ru` in the variables elsewhere in this `.pro` file. Note that you can install the Belurusian keyboard using `sudo apt install ubuntu-keyboard-belurusian` after `ubports-qa install keyboard-component 146` completes.